extensions/mqtt/processors/ConsumeMQTT.h

```diff
@@ -100,7 +100,7 @@ class ConsumeMQTT : public processors::AbstractMQTTProcessor {
   void onTrigger(const std::shared_ptr<core::ProcessContext> &context, const std::shared_ptr<core::ProcessSession> &session) override;
   // Initialize, over write by NiFi ConsumeMQTT
   void initialize(void) override;
-  bool enqueueReceiveMQTTMsg(MQTTClient_message *message) override;
+  virtual bool enqueueReceiveMQTTMsg(MQTTClient_message *message);
```

Comment:
this change is backwards; `override` is correct

Reply:
Updated.